app/room/[roomId]/page.js

```
@@ -0,0 +1,327 @@
+  const [messages, setMessages] = useState([]);
+  const [participants, setParticipants] = useState(1); // Start with just the local user
+  const [isConnected, setIsConnected] = useState(false);
+  const [copyLinkText, setCopyLinkText] = useState('Copy Link');
+  
+  const localVideoRef = useRef(null);
+  const remoteVideoRef = useRef(null);
+  const peerConnectionRef = useRef(null);
+  const localStreamRef = useRef(null);
+  const signalingRef = useRef(null);
+  
+  useEffect(() => {
```

Biome (1.9.4)

[error] This hook specifies more dependencies than necessary: roomId

This dependency can be removed from the list.


(lint/correctness/useExhaustiveDependencies)